### PR TITLE
add example of using a valid master element within an invalid master …

### PR DESCRIPTION
**Pull request by dericed**
This responds to the request for an example at the end of https://mailarchive.ietf.org/arch/msg/cellar/koyh6rLPjpqwmL9iUx2xouKD2g4.

### CODE REVIEW

File: specification.markdown [hunk 1]
```diff
@@ -864,6 +864,25 @@ The following scenarios describe events to consider when reading EBML Documents
...
+```xml
+<CONTACT>
+    <CRC-32>c119a69b</CRC-32><!- does not validate -->
+    <NAME>
+        <CRC-32>1f59ee2b</CRC-32><!- does not validate -->
+        <FIRST-NAME>invalid data</FIRST-NAME>
+        <LAST-NAME>invalid data</LAST-NAME>
+    </NAME>
+    <ADDRESS>
+        <CRC-32>df941cc9</CRC-32><!- validates -->
```
Comment (retokromer) [inline specification.markdown]: All three comments should start with `<!--` not `<!-`.

Comment (dericed) [reply specification.markdown]: updated

**Comment by robUx4** [comment]
LGTM, very good idea

**Comment by dericed** [comment]
I changed the language for clarity a little.

**Pull request merged**